.github/workflows/main.yaml

```diff
@@ -189,6 +226,66 @@ jobs:
           type: ondevice
           os: linux
 
+  # Runs on-host integration and unit tests.
+  on-device-test:
+    needs: [initialize, build]
+    if: |
+      needs.initialize.outputs.on_device_test == 'true' && (
```

Comment:
I don't see workflow_dispatch and schedule events handled here.

Reply:
Resolved